clang/lib/Sema/HLSLExternalSemaSource.cpp

```diff
@@ -290,8 +290,6 @@ struct BuiltinTypeDeclBuilder {
                                              SourceLocation()));
     MethodDecl->setLexicalDeclContext(Record);
     MethodDecl->setAccess(AccessSpecifier::AS_public);
-    MethodDecl->addAttr(AlwaysInlineAttr::CreateImplicit(
-        AST, SourceRange(), AlwaysInlineAttr::CXX11_clang_always_inline));
```

Comment:
Why was it necessary to remove this? I think we want this because we'll want these calls inlined even in separate compilation cases.

Reply:
We don't really have a workflow in Clang implemented yet for separate compilation. The analog in DXC would be something like `-T lib_6_x -default-linkage=external`. In that case we would still want these functions to be inlined, and we can accomplish that with this attribute.